script.js

```diff
@@ -126,3 +126,26 @@ shuffleButton.addEventListener('click', () => {
     );
   }
 });
+
+document.getElementById('start-quiz').addEventListener('click', () => {
+  const player1Name = document.getElementById('player1-name').value;
+  const player2Name = document.getElementById('player2-name').value;
+  if (player1Name && player2Name) {
+    document.getElementById('player1-display').textContent = player1Name;
+    document.getElementById('player2-display').textContent = player2Name;
+    document.getElementById('players-container').style.display = 'block';
+  } else {
+    alert('Please enter both player names!');
+  }
+});
+
+function updateScore(player, action) {
+  const playerPoints = document.getElementById(`${player}-points`);
+  let points = parseInt(playerPoints.value);
+  if (action === 'correct') {
+    points += 1;
+  } else if (action === 'wrong') {
+    points -= 1;
+  }
```

Comment:
In the exercise it is requested that when someone gets a question wrong (meaning we are clicking the Wrong option) the **other player gets a point**. Instead, here you are decreasing the player's own points. This moreover means that a player's points can become negative.